apps/full-site-editing/full-site-editing-plugin/lib/feature-flags/feature-flags.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * Feature flags file.
+ *
+ * @package full-site-editing
+ */
+
+/**
+ * Class A8C_Full_Site_Editing_Feature_Flags
+ */
+class A8C_Full_Site_Editing_Feature_Flags {
+	/**
+	 * Class instance.
+	 *
+	 * @var A8C_Full_Site_Editing_Feature_Flags
+	 */
+	private static $instance = null;
+
+	/**
+	 * Feature flags.
+	 *
+	 * @var array
+	 */
+	private $flags = array();
```

Comment:
What do you think about using array shorthands throughout this file? We specified min PHP at 5.6.20 for the plugin in the repo

Reply:
Sure, I like them more anyway. Updated in ca80013dca76a3546a24ecaf596636b074e25de7.